db/migrate/20160111162006_change_columns_in_users.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeColumnsInUsers < ActiveRecord::Migration
+  def change
+    change_column(:users, :first_name, :string, null: false)
+    change_column(:users, :last_name, :string, null: false)
+    remove_column(:users, :admin)
+  end
+end
```

Comment:
Have you tested this via `rake db:rollback`? `change_column` won't know what to roll back to, and neither will `remove_column` (it doesn't have information as to what the column was before, it doesn't know what type it was or whether it had any constraints)

This is a situation where you need to define separate `up/down` methods, rather than `change`, and be more specific about what's happening on rollback